src/components/Discounted-Products/DiscountedProducts.jsx

```diff
@@ -1,20 +1,25 @@
-import { discountedProductsArr } from "../../utils/filteredCategoriesArr";
+import { discountedProductsArr } from "../../utils/utilityArrays";
 import "./discountedProducts.css";
 import { Link } from "react-router-dom";
+import { MdOutlineDoubleArrow } from "../../utils/icons";
 
 const DiscountedProducts = () => {
     return (
         <div className="ap_homePage-discountedProducts-categories">
             <div className="ap_discountedProducts-scrollable-section">
-                 {
-                     discountedProductsArr.map((discountedProducts, index) => {
-                         return (
-                             <Link to="/Productlisting" key= { index } state= { discountedProducts }>
-                                 <img src= { discountedProducts.img } alt= "discounted Product" className="ap_discountedProduct-img" />
-                             </Link>
-                         )
-                     })
-                 }
+                <div className="ap_products-scrollable-section-description scrollable-section-description">
+                    <p>Latest grab offers only for you!</p>
+                    <MdOutlineDoubleArrow className="ap_products-scrollable-section-icon" />
+                </div>
+                {
+                    discountedProductsArr.map((discountedProducts, index) => {
+                        return (
```

Comment:
instead of writing return just use () brackets instead of curly {} arr.map(item=>(<div>item</div>)) no need to write return just a suggestion

Reply:
yes, we could have also done it in a way you have suggested, this didn't come to me earlier, thankyou for highlighting it.